androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/playerOverlay/Settings.kt

```diff
@@ -65,15 +77,71 @@ fun Settings(
     }
 }
 
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun TrackSelector(
```

Comment:
You don't need to create a new component just for the track selector. Note that the `SettingsSelector' is a generic component, so it should work for all cases where we need a selector. All you need to do is create a class that takes the `Pair<String, TrackSelectionParameters.Builder>` and then overrides the toString method.

Reply:
Done